undefined reference to `N_'

I'm getting the following error when building feedreader for rawhide:

```
CMakeFiles/feedreader.dir/src/Widgets/MediaPlayer.c.o: In function `feed_reader_media_player_toggleMute':
/builddir/build/BUILD/FeedReader-2.0.2/src/Widgets/MediaPlayer.vala:399: undefined reference to `N_'
CMakeFiles/feedreader.dir/src/Widgets/MediaPlayer.c.o: In function `_feed_reader_media_player_toggleMute_gtk_button_clicked':
MediaPlayer.c:(.text+0xc3): undefined reference to `N_'
CMakeFiles/feedreader.dir/src/Widgets/MediaPlayer.c.o: In function `_feed_reader_media_player_togglePause_gtk_button_clicked':
MediaPlayer.c:(.text+0xca7): undefined reference to `N_'
MediaPlayer.c:(.text+0xd33): undefined reference to `N_'
CMakeFiles/feedreader.dir/src/Widgets/MediaPlayer.c.o: In function `___lambda58__gasync_ready_callback':
MediaPlayer.c:(.text+0x1bc0): undefined reference to `N_'
CMakeFiles/feedreader.dir/src/Widgets/MediaPlayer.c.o:MediaPlayer.c:(.text+0x1cfd): more undefined references to `N_' follow
collect2: error: ld returned 1 exit status
```

It builds just fine on Fedora 25 though. Not sure what has changed. Perhaps different vala version plays a role? F25 has vala 0.34.4 while rawhide has vala 0.35.6.

Full build logs at https://koji.fedoraproject.org/koji/taskinfo?taskID=18092312

If I remove the N_, I get the same error on Fedora 25: `undefined reference to `_'`
So not sure what to do here. One way works for the old vala the other works for the new vala version.
Only solution I can imagine is to not make these strings translatable.

--- a/Constants.vala.in
+++ b/Constants.vala.in
@@ -43,11 +43,11 @@ namespace FeedReader {
 	}
 
 	public class MediaButton : GLib.Object {
-		public const string PLAY = N_("Play");
-		public const string PAUSE = N_("Pause");
-		public const string MUTE = N_("Mute");
-		public const string UNMUTE = N_("Unmute");
-		public const string CLOSE = N_("Close");
+		public const string PLAY = "Play";
+		public const string PAUSE = "Pause";
+		public const string MUTE = "Mute";
+		public const string UNMUTE = "Unmute";
+		public const string CLOSE = "Close";
 	}
 
 	public class Constants : GLib.Object {

